checks/code_review_test.go

```diff
@@ -229,7 +229,7 @@ func TestCodereview(t *testing.T) {
 				},
 			},
 			expected: checker.CheckResult{
-				Score: 0,
+				Score: checker.MaxResultScore,
```

Comment:
We automatically give full points if the commit was associated with a Phabricator revision even if there's no Reviewed-By. For now, everyone using Phabricator + GitHub will get a score bump - since this is only on the default branch, we already expect that those commits have a level of scrutiny even if they haven't been reviewed on Phabricator.